work-item-comments.go

```diff
@@ -92,21 +96,24 @@ func (c *WorkItemCommentsController) List(ctx *app.ListWorkItemCommentsContext)
 // Relations runs the relation action.
 // TODO: Should only return Resource Identifier Objects, not complete object (See List)
 func (c *WorkItemCommentsController) Relations(ctx *app.RelationsWorkItemCommentsContext) error {
+	offset, limit := computePagingLimts(ctx.PageOffset, ctx.PageLimit)
 	return application.Transactional(c.db, func(appl application.Application) error {
 		wi, err := appl.WorkItems().Load(ctx, ctx.ID)
 		if err != nil {
 			jerrors, _ := jsonapi.ErrorToJSONAPIErrors(goa.ErrUnauthorized(err.Error()))
 			return ctx.NotFound(jerrors)
 		}
 
-		comments, err := appl.Comments().List(ctx, ctx.ID)
+		comments, tc, err := appl.Comments().List(ctx, ctx.ID, &offset, &limit)
+		count := int(tc)
 		if err != nil {
-			jerrors, _ := jsonapi.ErrorToJSONAPIErrors(goa.ErrUnauthorized(err.Error()))
+			jerrors, _ := jsonapi.ErrorToJSONAPIErrors(goa.ErrInternal(err.Error()))
```

Comment:
You can replace these two lines with `return jsonapi.JSONErrorResponse(ctx, err)`